src/panels/config/integrations/ha-config-integration-page.ts

```diff
@@ -1259,6 +1283,15 @@ class HaConfigIntegrationPage extends SubscribeMixin(LitElement) {
     });
   }
 
+  private async _reconfigureIntegration(configEntry: ConfigEntry) {
+    showConfigFlowDialog(this, {
+      startFlowHandler: configEntry.domain,
+      showAdvanced: this.hass.userData?.showAdvanced,
+      manifest: await fetchIntegrationManifest(this.hass, configEntry.domain),
+      source: "reconfigure",
```

Comment:
Shouldn't we instead pass the current config entry id?

Reply:
How does this work with domains that accept multiple config entries?

Reply:
Probably not. I somehow forgot to include that. Thanks.